libs/partners/voyageai/langchain_voyageai/embeddings.py

```diff
@@ -37,8 +37,10 @@ class VoyageAIEmbeddings(BaseModel, Embeddings):
     _aclient: voyageai.client_async.AsyncClient = PrivateAttr()
     model: str
     batch_size: int
+
+    output_dimension: Optional[Literal[256, 512, 1024, 2048]] = None
     show_progress_bar: bool = False
-    truncation: Optional[bool] = None
+    truncation: bool = True
```

Comment:
is changing this default a breaking change for any possible usage of `langchain-voyageai` prior to this PR?

Reply:
No - it seems was `True` anyway - https://github.com/voyage-ai/voyageai-python/blob/012d16bd708c797766beb26555e56b10db8d09fd/voyageai/client.py#L51